node_modules/oae-core/about/css/about.css

```diff
@@ -0,0 +1,62 @@
+
+#about-description .about-description-empty,
+#about-description .about-description-empty a {
+    color: #888;
+}
+
+#about-description .about-description-empty a {
+    font-style: italic;
+    text-decoration: underline;
+}
+
+#about-members {
+    margin-top: 35px;
+}
+
+#about-members > div.about-show-all {
+    margin-top: 15px;
+}
+
+#about-members .about-members {
+    display: -webkit-box;
+    display: -moz-box;
+    display: -ms-flexbox;
+    display: -webkit-flex;
+    display: flex;
+    width: 180px;
+    -webkit-flex-wrap: wrap;
+        -ms-flex-wrap: wrap;
+            flex-wrap: wrap;
+}
+
+#about-members .about-members div.oae-thumbnail {
+    border: none;
+    height: 60px;
+    width: 60px;
+    -webkit-box-flex: none;
+       -moz-box-flex: none;
+        -webkit-flex: none;
+            -ms-flex: none;
+                flex: none;
+}
+
+/* Clear the border radius on base oae-thumbnail style */
+#about-members .about-members div.oae-thumbnail,
+#about-members .about-members div.oae-thumbnail a,
+#about-members .about-members div.oae-thumbnail div {
+    -webkit-border-radius: 0;
```

Comment:
According to http://caniuse.com/#feat=border-radius, it should be safe to now just use `border-radius`. If we have any other place where we're still setting all of the unnecessary prefixed versions, we can remove those as well.